src/lib.rs

```diff
@@ -0,0 +1,101 @@
+/*###############################################################
+###                                                             #
+###                        KVStore Code                         #
```

Comment:
Given this would be a multi-file thing. Should we make this part of the REPL/CLI?

Reply:
Also call this `stylus`?